scripts/jenkins/jobs-obs/templates/openstack-upstream-gerrit-rpm-packaging.yaml

```diff
@@ -53,6 +46,15 @@
           osc status
           osc commit -m "rpm-packaging CI (${{ZUUL_CHANGES}})"
 
+          # also create links for new packages
+          echo "creating missing links ..."
+          for x in *-SLE_12_SP1.spec; do
+              pkgname=${{x//-SLE_12_SP1.spec/}}
```

Comment:
I don't care which one, I just kept it the same like in the upstream-gerrit-rpm-packaging job. if we change we need to change it in both places though.